config/initializers/teaspoon.rb

```diff
@@ -0,0 +1,5 @@
+unless Rails.env.production?
+  class Teaspoon::SuiteController < ActionController::Base
+    skip_before_action :handle_two_factor_authentication
```

Comment:
is there a way to include this somewhere in the `spec` directory instead? I would generally not expect to find code for the specs in `config` or `app` but I have not set up teaspoon before!